tensilelite/Tensile/bin/Tensile

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/env python3
-
+#!/bin/bash
```

Comment:
Changed from Python to shell script. CMake and Python is seperated and the module path is pased to the Python though PYTHONPATH.